servers/src/main/java/tachyon/master/permission/FileSystemPermissionChecker.java

```diff
@@ -49,7 +50,63 @@ public static synchronized void initializeFileSystem(boolean permissionCheckEnab
   }
 
   /**
-   * This method checks access permission.
+   * Checks requested permission on the last node and basic permission on the whole path.
```

Comment:
REPLY UPDATES:
I am developing TACHYON-1418, seen as the caller of these methods. Then I found it is trivial for caller to cut the path if checking parent permission. It seems ok to only use `checkPermission` and let caller to decide whether self or parent case. Then the api is clear and not cause confusion.

Let me know your thoughts and I could update patch with keeping only one API in next commit if agreed.